packages/KIcon/icons/index.js

```diff
@@ -40,12 +40,15 @@ import forward from './icn-forward.svg'
 import gateway from './icn-gateway.svg'
 import gear from './icn-gear.svg'
 import gearFilled from './icn-gear-filled.svg'
+import getStarted from './icn-get-started.svg'
 import grid from './icn-grid.svg'
 import help from './icn-help.svg'
 import immunity from './icn-immunity.svg'
 import increase from './icn-increase.svg'
+import introduction from './icn-introduction.svg'
 import info from './icn-info.svg'
 import kong from './icn-kong.svg'
+import kongLearning from './icn-kong-learning.svg'
```

Comment:
I think we need to name these icons what they ARE, instead of the name that marketing refers to them as.

e.g. "flag" instead of "introduction", etc.

Reply:
Done!